pytket/pytket/passes/superpass.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019-2023 Cambridge Quantum Computing
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+from typing import Optional, Callable
+
+from pytket.circuit import Circuit
+from .._tket.passes import BasePass
+
+
+class superpass:
```

Comment:
I think you need to add this to `pytket/docs/passes.rst` for the documentation to be picked up by sphinx.